baus/postprocessing.py

```diff
@@ -336,8 +346,17 @@ def GEO_SUMMARY_LOADER(run_num, geo, parcel_baseyear, parcel_endyear):
 ##Similar to above, this is to define a separate fileloader to produce summaries for overlapping geographies. W
 def TWO_GEO_SUMMARY_LOADER(run_num, geo1, geo2, parcel_baseyear, parcel_endyear):
 
+    if 'fbpchcat' in parcel_endyear.columns:
+      zoningtag = 'fbpchcat'
+    elif 'pba50chcat' in parcel_endyear.columns:
+      zoningtag = 'pab50chcat'
+    elif 'zoningmodcat' in parcel_endyear.columns:
+      zoningtag = 'zoningmodcat'
+    else: 
+      zoningtag = 'zoninghzcat'
```

Comment:
@bobbylu I left a comment on the counterpart to this in BAUS, and can update you if it gets updated!: https://github.com/BayAreaMetro/bayarea_urbansim/pull/236/files#diff-a06ab55985dd3575034bf5c24d0959c52471ec4756ea987c31ef859ee7616649L1259

Reply:
@bobbylu I'm going to remove the summaries.py part of this from the Inclusionary PR in order to move it along. I think it belongs in this PR so could you add it:
https://github.com/BayAreaMetro/bayarea_urbansim/pull/236/files#diff-a06ab55985dd3575034bf5c24d0959c52471ec4756ea987c31ef859ee7616649L1259

Reply:
I'm not sure if it's linking well but it should be line 1259. The comments are:

@theocharides theocharides 21 hours ago Member
@yuqiww would this be an elif, so that only Horizon scenarios use zoninghmodcat? I think you could also switch them, and else defaults to zoningmodcat, since PBA40 used to be default scenario, but that might be less ideal.

These scenario handlers were originally used to turn things on and off, so you could remove a scenario from the list for example, and it would turn off the upzoning for testing. I updated their in-line comments recently to reflect that their multi-purpose now, so they shouldn't be used for that function. So maybe we should add to those comments that these also control post processing summaries now.

@yuqiww yuqiww 17 hours ago Author Member
Re the first point, yes, you are right, I just wasn't sure which zoningmod applies to which plans.
Re the second point, adding the comment sounds great, unless you prefer setting new scenario handlers for summarizing. But we are already using the current handlers in topsheet() and geographic_summaries().